packages/theme-ui/src/jsx.js

```diff
@@ -3,7 +3,8 @@ import css from '@styled-system/css'
 
 const getCSS = props => theme => {
   if (!props.sx && !props.css) return undefined
-  const styles = css(props.sx)(theme)
+  const sx = Array.isArray(props.sx) ? props.sx : [props.sx]
+  const styles = sx.map(s => css(s)(theme))
   const raw = typeof props.css === 'function' ? props.css(theme) : props.css
   return [styles, raw]
```

Comment:
Should `styles` also be spread into the array here now?

Reply:
Not quite sure what you mean, `styles` is passed to emotions `css` prop which accepts an array and does the actual composition

Reply:
With your change, this would be passing a nested array along to emotion – e.g. `[ [ a, b ], c ]`

Reply:
Hi @jxnblk, What value as an `sx` prop would pass a nested array to emotion? Could you add a failing test?

Reply:
Hi @jxnblk, I have spread the `styles` in the return array